src/containers/User/Profile/Profile.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import Box from "@material-ui/core/Box";
+import { ContactInfo } from "../../../components/ContactInfo/ContactInfo";
+import UpdatePassword from "../../../components/UpdatePassword/UpdatePassword";
+import "./Profile.scss";
+import Copyright from "../../../components/CopyRight";
+import Container from "@material-ui/core/Container";
+import DeactivateAccount from "../../../components/DeactivateAccount/DeactivateAccount";
+import { TabPanel } from "../../../components/TabPanal/TabPanel";
+
+function additionalProps(index) {
+  return {
+    id: `vertical-tab-${index}`,
+    "aria-controls": `vertical-tabpanel-${index}`,
+  };
+}
+
+const user = {
+	name: "Abdullah",
+	email: "abdullah@email.com",
+	phone: "0123456789",
+	address: "Aleandria, Egypt",
+	gender: "male",
+	profileImg: "https://via.placeholder.com/150",
+	dob: "1994-7-10"
+};
+
+export default function VerticalTabs() {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+		<Container component="main">
```

Comment:
You need to use proper linter. You can use ```ESLint```

Reply:
Unfortunately, I have troubles installing it.